api/v1alpha5/virtualmachine_storage_types.go

```diff
@@ -354,25 +354,40 @@ type VirtualMachineStorageStatus struct {
 	Used *VirtualMachineStorageStatusUsed `json:"usage,omitempty"`
 }
 
+type VirtualMachineStorageStatusUsedSnapshotDetails struct {
+	// +optional
+
+	VM *resource.Quantity `json:"vm,omitempty"`
+
+	// +optional
+
+	Volume *resource.Quantity `json:"volume,omitempty"`
```

Comment:
Please document these fields. The example I sent you did not have docs because you were intended to write them as mine was just an example :)